artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQMessageBundle.java

```diff
@@ -551,4 +551,7 @@ IllegalStateException invalidRoutingTypeUpdate(String queueName,
 
    @Message(id = 229253, value = "Unable to acquire OperationContext when replicating packet: {}. ExecutorFactory: {}")
    IllegalStateException replicationFailureRepliTokenNull(String packet, String executorFactory);
+
+   @Message(id = 229254, value = "{} must be a power of 2 (actual value: {})")
```

Comment:
This class is conflicting, presumably some other new message/id addition was made here so this one will need bumped accordingly.

Reply:
Done